gst/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -388,58 +443,38 @@ TFCore::run (const GstTensorMemory * input, GstTensorMemory * output)
   std::vector<Tensor> outputs;
   int array_len;
 
-  for (int i = 0; i < inputTensorMeta.num_tensors; i++) {
-    TensorShape ts = TensorShape({});
-    for (int j = inputTensorRank[i] - 1; j >= 0; j--){
-      ts.AddDim(inputTensorMeta.info[i].dimension[j]);
+  for (int i = 0; i < inputTensorMeta.num_tensors; ++i) {
+    TensorShape tensorShape = TensorShape({});
+    for (int j = inputTensorRank[i] - 1; j >= 0; --j){
+      tensorShape.AddDim(inputTensorMeta.info[i].dimension[j]);
     }
-    Tensor inputTensor(
-      getTensorTypeToTF(input->type),
-      ts
-    );
 
-    array_len = input->size / tensor_element_size[input->type];
+    TFBuffer *buf = new TFBuffer;
+    buf->len_ = input->size;
```

Comment:
input is array of tensors, please fix this.
input[i].data / input[i].size / input[i].type